emulators/mame/Portfile

```diff
@@ -125,9 +159,15 @@ depends_run-append  \
 # Build Options
 #------------------------------------------------------------------------------
 
-use_parallel_build  yes
+# Note: ARM64 support untested
+#supported_archs     ppc ppc64 i386 x86_64 arm64
+supported_archs     ppc ppc64 i386 x86_64
 
+use_parallel_build  yes
```

Comment:
This being the default, we don't usually bother to add it, as it's just clutter.

Reply:
No problem, I'll remove.